index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>HTML & CSS Training</title>
+    <meta name="description" content="training" />
+    <meta name="author" content="Wolox" />
+    <link rel="stylesheet" type="text/css" href="application.css" />
+  </head>
+  <body>
+    <div class="book-container">
+      <img class="book-cover" src="assets/book-cover.png" />
```

Comment:
`img` tags always have to have an `alt` attribute --> `<img class="book-cover" src="assets/book-cover.png" alt="book cover"/>`